File: extra.py
import time
import random

NUMEROUSUARIOS = 10000

def cadastramento_completo(usuarios):
    for i in range(NUMEROUSUARIOS):
        if i not in usuarios:
            return False
    return True

def algoritmo1():
    usuarios = []

    for i in range(NUMEROUSUARIOS):
        
        while True:
            id_aleatorio = random.randrange(NUMEROUSUARIOS)
            if id_aleatorio not in usuarios:
                usuarios.append(id_aleatorio)
                break

    #print (usuarios)
    #print (cadastramento_completo(usuarios))
                

def algoritmo2():
    pass


comeco = time.time()
algoritmo1()
fim = time.time()

duracao = fim - comeco

print (f"{duracao:.4f}")

File: extra.cpp
#include <chrono>
#include <iostream>
#include <iomanip>
#define USUARIOS 100

using namespace std;

void algoritmo1() {
    int usuarios[USUARIOS];

}

void algoritmo2() {
    
    
}


int main(int argc, char const *argv[])
{
    auto comeco = chrono::steady_clock::now();
    algoritmo1();
    auto fim = chrono::steady_clock::now();

    double duracao = chrono::duration_cast<chrono::milliseconds>(fim - comeco).count() / 1000.0;   

    cout << fixed << setprecision(4) << duracao << endl;

    return 0;
}
